files/default/sql/ddl/updates/2.5.0.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE `hopsworks`.`validation_rule` MODIFY COLUMN description VARCHAR(200) CHARACTER SET latin1 COLLATE latin1_general_cs DEFAULT NULL;
+ALTER TABLE `hopsworks`.`validation_rule` DROP INDEX `unique_validation_rule`;
+ALTER TABLE `hopsworks`.`validation_rule` ADD CONSTRAINT `unique_validation_rule` UNIQUE KEY (`name`);
```

Comment:
This migration may fail if we have entries in the table with the same `name`, but as this is the validation_rule description table populated by us it should not happen right?

Reply:
Correct, the tables is not accessible by users and it does not contain duplicates.